PDSA_System/Client/Pages/Login.razor

```diff
@@ -51,6 +51,9 @@
                 var token = task.Result.Content.ReadAsStringAsync().Result;
 
                 LocalStorage.SetItemAsStringAsync("token", token);
+                //Linje nedenfor brukes av TeamlederPage. Må endres manuelt foreløpig. Teams må inn i local storage.
+                //Endre string data for å endre team.
+                LocalStorage.SetItemAsStringAsync("teamId", "4");
```

Comment:
Skal dette være her?

Reply:
Ja. Dette setter TeamId i session. Kan forklare nærmere senere.

Reply:
Men hva skjer hvis min TeamID er 2 da? Tallet 4 er jo hardkodet

Reply:
For å implementere dette må det teamid inn i tokenet som opprettes ved pålogging. Endringer må gjøres på authcontroller og Login razorpage.

Er på litt dypt vann her, så mulig terminologien min er noe feil her.

Reply:
> For å implementere dette må det teamid inn i tokenet som opprettes ved pålogging. Endringer må gjøres på authcontroller og Login razorpage.

Jeg hadde ikke satt det inn i token, da en endring av dette vil invalidere det og logge ut brukeren. Gjør det heller mulig for bruker å se en enkel liste over team, og legg til en knapp for å endre team, så vil den redigere på den nye verdien som du har opprettet